packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -72,11 +71,13 @@ class FlutterDevice {
     if (vmServices != null)
       return;
     final List<VMService> localVmServices = List<VMService>(observatoryUris.length);
-    for (int i = 0; i < observatoryUris.length; i++) {
+    for (int i = 0; i < observatoryUris.length; i += 1) {
```

Comment:
Curious - what's wrong with `i++` in a for loop?

Reply:
Nothing, I just have a personal vendetta against `++`.